lib/fulltext.js

```diff
@@ -6,13 +6,6 @@ exports.data_records = {
       return null;
```

Comment:
This is the definition file for the lucene indexes. My change means you can't search for, for example, reports by clinic. Lucene searches are hopefully on the way out anyway...